data-modeling.md

```diff
@@ -6,19 +6,253 @@ For these exercises, create a Database Schema Diagram for the following scenario
  - <code>https://sqldbm.com/</code>
  - <code>https://app.genmymodel.com/</code>
 
-1. TEKCafe  is looking to design a relational database management system.  They have to keep track of their customers in relation to the orders they make.  TEKCafe is interested in keeping track of all of their products that their customers will order.  TEK Cafe currently has the following products available for customers to purchase: 
+1. TEKCafe  is looking to design a relational database management system.  They have to keep track of their customers in relation to the orders they make.  TEKCafe is interested in keeping track of all of their products that their customers will order.  TEK Cafe currently has the following products available for customers to purchase:
 
 - Teas
 - Coffees
 - Coffee Mugs
 - Frappuccinos
-- Design a relational database schema to keep track of the data that TEK Cafe is looking to model.  
+- Design a relational database schema to keep track of the data that TEK Cafe is looking to model.
+
+CREATE TABLE `Customers` (
+	`customer_id` INT NOT NULL AUTO_INCREMENT,
+	`customer_name` varchar(255) NOT NULL,
+	`customer_email` varchar(255) NOT NULL,
+	PRIMARY KEY (`customer_id`)
+);
+
+CREATE TABLE `customer_orders` (
+	`customer_order_id` INT NOT NULL AUTO_INCREMENT,
+	`customer_id` INT NOT NULL,
+	PRIMARY KEY (`customer_order_id`)
+);
+
+CREATE TABLE `products` (
+	`product_id` INT NOT NULL AUTO_INCREMENT,
+	`product_name` varchar(50) NOT NULL,
+	`product_description` TEXT NOT NULL,
+	`price` DECIMAL NOT NULL,
+	`product_type` varchar(50) NOT NULL,
+	PRIMARY KEY (`product_id`)
+);
+
+CREATE TABLE `order_items` (
+	`order_item_id` INT NOT NULL AUTO_INCREMENT,
+	`customer_order_id` INT NOT NULL,
+	`product_id` INT NOT NULL,
+	`product_quantity` INT NOT NULL,
+	`order_price` INT NOT NULL,
```

Comment:
NOTE: not sure if mySQL has it, but this could be a calculated column or have a trigger for when the quantity changes (but having it handled in code has some advantages, too)